src/core/ngx_connection.c

```diff
@@ -487,8 +487,17 @@ ngx_open_listening_sockets(ngx_cycle_t *cycle)
                 continue;
```

Comment:
For the commit message of `Core: added socket protocol`:

- usually, we use the present form, but looking at the history here, they seem to do like you, except that they always add a `.` at the end (no idea why, but best to do the same)
- Start your sentences with a capital letter: `structure. the`
- What is very important in a commit message is to add the reason why this commit is there: we can easily see what you modified, understand why you did that is harder. You can add something:
```
Modules will then be able to specify a different protocol, e.g.
IPPROTO_MPTCP.
```